qemu/tests/balloon_stress.py

```diff
@@ -51,8 +56,15 @@ def run_stress(test, params, env, vm):
         balloon_test = BallooningTestLinux(test, params, env)
 
     error_context.context("Run stress background", logging.info)
-    bg = utils_misc.InterruptedThread(run_stress, (test, params, env, vm))
-    bg.start()
+    if params['os_type'] == 'windows':
+        video_player = utils_misc.set_winutils_letter(session, params["mplayer_path"])
+        video_url = params["video_url"]
+        play_video_cmd = params["play_video_cmd"] % (video_player, video_url)
+        session.cmd(play_video_cmd)
+        time.sleep(5)
```

Comment:
How about wait_for check_bg_running ?

Reply:
I'm not sure if it is necessary. wait_for means we will keep checking the process until it is running. But IMHO, we don't want to wait every time we send a cmd to guest. I would prefer to check the process only once before doing next test instead of waiting for it. Thanks!

Reply:
So, what's the sleep here used for ?   It is used to wait for the stress is running after loaded (only one time) then begin to do balloon, right ?  I think wait for is more reliable than sleep.

Even use `wait_for`, we only check once before doing next step, not every time.

The latest update use `sleep` for both linux and windows.  For linux guest, `load_stress_tool` already wait for the stress running.  I think it is redundant for linux guest.

Reply:
@yanan-fu If we only check once, why do we need wait_for ? It might be redundant for linux guest to do extra sleep , but I think it makes code more readable  to put it before check_bg_running instead of only putting it in the block for windows. Thanks!

Reply:
@huangyum  your `once` is not what i mean. I said `once`, it means we only use `wait_for` one time to wait for the stress begin running.  I don't care how many times it checking the process in func `wait_for` until time out.  This is what `wait_for` used for.

Reply:
@yanan-fu Thanks, have updated to use wait_for, please help review.  Thanks!